### PR TITLE
Simplify setup - automatically figure out artifact paths

### PR DESCRIPTION
**Pull request by mikrostew**
Make the build and install process less painful by automatically figuring out the paths to the build artifacts.

### CODE REVIEW

File: support/unix/build.sh [hunk 1]
```diff
@@ -4,19 +4,16 @@ usage() {
...
 
 The output file is saved as ./install.sh.
 END_USAGE
 }
 
-if [ "$#" -ne 4 ]; then
-  usage >&2
-  exit 1
+target_dir='release'
+if [ -n "$1" ]; then
```
Comment (dherman) [inline support/unix/build.sh]: Can you also add a test that if `$#` is not 0 or 1, and if `$1` is neither `release` nor `debug` nor empty, that it should call `usage()` and `exit 1`?

Comment (mikrostew) [reply support/unix/build.sh]: Good call I'll fix that
